### DISCUSSION TITLE
Steam Deck Canary Update Problem

### DISCUSSION THREAD

**Discussion by M45TR0** [tsunamods-codes/7th-Heaven#207] [Q&A] [answered]
Every time I try to update to 3.9.9.28 on my Steam Deck, the batch file for the update starts and I get an error "Internal error: Failed to expand shell folder constant "userpf"".

Afterwards I tried to update to 3.5.1.0 and it worked without a problem, but now I'm unable to get back to the canary builds.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Thanks for rising this discussion but we do not offer official support on the repository for Steam Deck as it's maintained elsewhere, see https://github.com/dotaxis/7thDeck

**Comment by dotaxis** [comment]
I think changes to the Inno setup will be necessary to fix this. I'm not sure what the cause is but stable installs while canary does not.

Looking into this but maybe we should reopen the issue.

Edit: https://github.com/tsunamods-codes/7th-Heaven/pull/215
